modules/custom/movie_reservation_controller/src/Controller/MovieReservationController.php

```diff
@@ -11,68 +11,107 @@ class MovieReservationController {
-        ];
+      $vid = 'movie_type';
+      
+      $terms =\Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadByProperties(['vid' => $vid]);
+
+      return [
+          '#theme' => 'article_list',
+          '#items' => $terms,
+          '#title' => 'Welcome to our movie reservation page'
+      ];
         
     }
 
     public function filter($movie_type){
 
-        $termIds = (array) $movie_type;
-        if(empty($termIds)){
-          return NULL;
-        }
+      $termIds = (array) $movie_type;
+      if(empty($termIds)){
+        return NULL;
+      }
 
-        $result = [];
-        $query = \Drupal::database()->select('taxonomy_index', 'ti');
-        $query->fields('ti', array('nid'));
-        $query->condition('ti.tid', $termIds, 'IN');
-        $query->distinct(TRUE);
-        $nodeId = $query->execute();
-      
-        if($nodeIds = $nodeId->fetchCol()){
-          $values =  \Drupal\node\Entity\Node::loadMultiple($nodeIds);
+      $result = [];
+      $query = \Drupal::database()->select('taxonomy_index', 'ti');
+      $query->fields('ti', array('nid'));
+      $query->condition('ti.tid', $termIds, 'IN');
+      $query->distinct(TRUE);
+      $nodeId = $query->execute();
+    
+      if($nodeIds = $nodeId->fetchCol()){
+        $values =  \Drupal\node\Entity\Node::loadMultiple($nodeIds);
 
-        }
+      }
 
-        foreach ($values as $value) {
-          
-          
-          $result[] = [
-            "id" => $value->id(),
-            "title" => $value->title->getString(),
-            "genre" => $value->field_movie_type->getString(),
-            "reservation_period" =>$value->field_reservation_period->getString()
-          ];    
-        }
+      foreach ($values as $value) {
+        
+        
+        $result[] = [
+          "id" => $value->id(),
+          "title" => $value->title->getString(),
+          "genre" => $value->field_movie_type->getString(),
+          "reservation_period" =>$value->field_reservation_period->getString()
+        ];    
+      }
 
-        return new JsonResponse([ 'data' => $result, 'method' => 'GET', 'status'=> 200]);
+      return new JsonResponse([ 'data' => $result, 'method' => 'GET', 'status'=> 200]);
 
-    }
+    } 
 
     public function save($customer_name,$movie){
 
-          $movie_data = explode("|", $movie);
-      
-          $connection = \Drupal\Core\Database\Database::getConnection();
-          $connection->insert('reservations')->fields([ 'day_of_reservation' => $movie_data[3],'time_of_reservation' => date('Y-m-d h:i:s'), 'reserved_movie_name' => $movie_data[1], 'reserved_movie_genre' => $movie_data[2], 'customer_name' => $customer_name ]) ->execute();
+      $movie_data = explode("|", $movie);
+
+      $database = \Drupal::database();
+      $query = $database->query('SELECT * FROM reservations WHERE reserved_movie_name = :reserved_movie_name AND day_of_reservation = :day_of_reservation', [
```

Comment:
Linija 65 je predugacka, neki coding standardi (koji se uglavnom primenjuju) se vode da u jednoj liniji treba imati do 120 karaktera, ne vise. Ovo mozes da primenjujes, a i ne moras, ovo ti je moj savet.

Mozes recimo da je formatiras da svaki uslov kverija bude jedan ispod drugog, isto ljudi dosta primenjuju to, jer im je lakse videti sta je uradjeno, nego da idu sa skrolom u editoru na desno. ;)